pom.xml

```diff
@@ -364,7 +364,7 @@
 			<dependency>
 				<groupId>xerces</groupId>
 				<artifactId>xercesImpl</artifactId>
-				<version>2.8.0</version>
+				<version>2.9.0</version>
```

Comment:
I tested the following versions: 2.9.1,2.10.0,2.11.0,2.12.0 all had build failures  but 2.9.0 passed cc @dkayiwa

Reply:
Did you share this as a comment on the ticket?